routes/users.js

```diff
@@ -0,0 +1,9 @@
+var express = require('express');
```

Comment:
you can remove this file if youre not going to use it --- then you would have to remove the appropriate functions in app.js --- app.use('/users', users) and --- var users = require('./routes/users');